google/cloud/storage/testing/storage_integration_test.cc

```diff
@@ -122,12 +122,7 @@ std::unique_ptr<RetryPolicy> StorageIntegrationTest::TestRetryPolicy() {
 }
 
 std::string StorageIntegrationTest::MakeRandomBucketName() {
-  // The total length of this bucket name must be <= 63 characters,
-  char constexpr kPrefix[] = "cloud-cpp-testing-";  // NOLINT
-  auto constexpr kMaxBucketNameLength = 63;
-  static_assert(kMaxBucketNameLength > sizeof(kPrefix),
-                "The bucket prefix is too long");
-  return testing::MakeRandomBucketName(generator_, kPrefix);
+  return testing::MakeRandomBucketName(generator_, "cloud-cpp-testing-");
```

Comment:
Vague suggestion:  We might like the trailing dash to be an underscore given than `testing::MakeRandomBucketName()` adds `"YYYY-MM-DD_"`.

Reply:
I like it.  I made that change in this pass, I will clean this further in the next pass (I am putting it off because I will need to change dependencies and I want that to be "pure" refactoring).

Reply:
Argh, that won't work...